src/data_analysis/symmetry_checker.py

```diff
@@ -0,0 +1,239 @@
+import re
```

Comment:
i was taught to formulate commit messages in the present tense. i think this is a google guideline

Reply:
I didn't know. Thanks for the hint